src/plugins/workspace/server/plugin.ts

```diff
@@ -108,6 +109,26 @@ export class WorkspacePlugin implements Plugin<WorkspacePluginSetup, WorkspacePl
     });
   }
 
+  private setUpRedirectPage(core: CoreSetup) {
+    core.http.registerOnPostAuth(async (request, response, toolkit) => {
+      const path = request.url.pathname;
+      if (path === '/') {
+        const listResponse = await this.client?.list(
```

Comment:
Nit: Prefer using a meaningful variable name, such as `workspaceListResponse`

Reply:
Sure, I will update the name